### ISSUE TITLE
Hiding series via its hide() method does not "gray out" respective legend item

### ISSUE THREAD

**Issue by martynasma** [amcharts/amcharts4#129]
```
chart.events.on("inited", function() {
  series2.hide();
});
```

This correctly hides series on the chart itself, but its legend item remains as if it was still visible - not grayed out.

Example: https://codepen.io/team/amcharts/pen/6937fc8450019234d09bd79ab07580b4?editors=0010

**Comment by martynasma** [comment]
Related issues: #70 and #129.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.37](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.37).

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
